filenames.gypi

```diff
@@ -64,6 +66,7 @@
       'lib/renderer/web-view/web-view-constants.js',
       'lib/renderer/api/desktop-capturer.js',
       'lib/renderer/api/exports/electron.js',
+      'lib/renderer/api/module-list.js',
```

Comment:
This should go after `'lib/renderer/api/ipc-renderer-setup.js',` to keep alphabetical order here.